risk-assessments.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: Risk Assessments
+date: 2022-04-16
+---
+
+Last updated: {{ page.date | date: "%e %B %Y" }}.
+
+## SR2022
+
+### Competition
+The risk assessment for the SR2022 Competition can be found [here]({{ 'resources/sr2022/risk-assessments/competition.pdf' | prepend: site.baseurl }}).
```

Comment:
nit: using "here" as link text is discouraged for accessibility reasons.

This link also doesn't work due to a missing `/` in the url.

```suggestion

We have published a [risk assessment for the SR2022 Competition]({{ '/resources/sr2022/risk-assessments/competition.pdf' | prepend: site.baseurl }}).
```